### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following: Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by HorselessName** [community/community#68551] [GitHub Education]
### Hi everyone,

I am trying to apply for a few weeks now but everytime I try there is a new error.
This time I am getting:

Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.

What do I do ?

### A suggestion...(optional)

Make it easier for students to apply for the pack.

**Comment by Primus66** [comment]
Use New Proof: If you’ve used a particular document or ID in a previous application, try using a different document or proof of enrollment for your current application. This can help avoid any conflicts with previous applications.

**Comment by rahulkbharti** [comment]
there are no diffrent user id provided bro...

**Comment by adilshaikh165** [comment]
Hello @HorselessName brother,

Even I'm facing the same issue which you faced related with proof has already been used.
Do you got any solution out of it?

Kindly help me with the same issue.

Thanks.
